common-components/index.md

```diff
@@ -186,20 +215,61 @@ __Further guidance__
+
+##### Opportunities for teams across DFE
+The Enterprise CRM service is constantly evolving, with new capabilities and functionality being incorporated regularly. Teams across DfE are adopting the Enterprise CRM services to:
+- enhance their current CRM solutions
+- benefit from more integration and data sharing
+- make significant savings on development and support
+
+##### Key benefits
+Below is an overview of some of the key benefits.
+
+**Strategic benefits:**
+- Aligns with DfE and wider government strategy
+- The Enterprise CRM strategic roadmap has a host of capability enhance-ments to increase functional maturity
+- System and development knowledge is retained in-house, enabling quicker and more flexible delivery of future capabilities
+
+**Technical benefits:**
+- Ready to use solution, with no procurement required
+- Over time, there will be increased opportunities to integrate with other DfE systems and improve data sharing
+- There’s no obligation to move from your current platform, the Enterprise CRM services can help enhance and evolve your current solution
+
+**Support benefits:**
+- Specialist in-house engineers delivering 2nd and 3rd line support
+- Operating model integrated with the ServiceNow toolset
+- Flexible and fast-turnaround change management
+- Testing and training for technical and user readiness
+
+**Cost benefits:**
+- Save money by consolidating support models, exiting from expensive exter-nal supplier contracts, and sharing central resources
+- Lowered cost of total ownership and economies of scale as multiple business areas contribute to maintenance costs
+
+##### Key documents
+Read the [Operating Model](https://educationgovuk.sharepoint.com/:w:/r/sites/lveesfa00073/_layouts/15/Doc.aspx?sourcedoc=%7b5E35BFD1-44C0-478B-81AA-368CAEED4CC3%7d&file=SharedCRM_Service_Operating_Model%20v0.1.docx&action=default&mobileredirect=true&CID=D33D9D75-C993-40C8-AAF5-449A95931FD9&wdLOR=c43C81DA6-1EF4-4B25-B4E7-F798DB011FE5) for more information about the solution design, security, availability, support and governance. If you need access to this document, please contact Matt Morgan.  
+
+##### Support
+The Enterprise CRM is supported internally by a central team within CIPS.  The team uses the ServiceNow toolset to manage the support of this service and follow DfE’s central service management processes.
+
+Contact the service desk to report an issue, the team are available from 08:00 – 22:00 hrs Monday to Friday, excluding English public holidays.  If the service desk is unable to resolve your query, the issue will be passed to a specialist technical team.  
+
+##### Roadmap
+The Enterprise CRM is tailored to meet unique customer requirements in every new deployment. A range of enhanced solution features are also planned, these include:
+- email
```

Comment:
Upper case E on email. These points are very vague. A CRM system that doesn't do email or document storage...?